src/pages/GetInvolved/GetInvolved.js

```diff
@@ -1,42 +1,46 @@
 import React from "react";
-import { useSelector } from "react-redux"
+import { useSelector } from "react-redux";
 import Layout from "components/Layout/Layout";
 import { Link } from "react-router-dom";
-import './GetInvolved.scss';
+import "./GetInvolved.scss";
 
 const GetInvolved = () => {
+  const clubDataAll = useSelector((state) => state.clubs);
 
-  const clubDataAll = useSelector(state => state.clubs)
-
-  const renderClubs = clubs => {
-    return clubs.map((club) => {
-      return (
-        <li>
-          <Link className="get-involved__items--link" key={club.id} to={club.link}>
-            {club.title.rendered}
-          </Link>
-        </li>
-      )
-    })
-  }
+  const renderClubs = clubDataAll.map((club) => {
+    return (
+      <li>
+        <Link className="get-involved__item-link" key={club.id} to={club.link}>
+          {club.title.rendered}
+        </Link>
+      </li>
+    );
+  });
 
   return (
     <Layout>
       <div className="get-involved__container">
-          <div className="get-involved__title">
-            <h2>HUOA Club Listing</h2>
-          </div>
-          <div className="get-involved__description">
-            <p2>Please contact HUOA if you are interested in joining and finding out more about the HUOA clubs.  Contact us at 676-5400.</p2>
-          </div>
+        <div className="get-involved__title">
+          <h2>HUOA Club Listing</h2>
+        </div>
+        <div className="get-involved__description">
+          <p>
+            Please contact HUOA if you are interested in joining and finding out
+            more about the HUOA clubs. Contact us at 676-5400.
+          </p>
+        </div>
```

Comment:
This needs to be connected to the backend data we have a clubs page in out getPages store. Please add the content from there. You should be able to pass the data down from routes like our donate page. 

Follow the data here: 

https://github.com/Hawaian-United-Okinawa-Association/huoa-react/blob/main/src/config/routes.js
https://github.com/Hawaian-United-Okinawa-Association/huoa-react/blob/main/src/pages/Donate/Donate.js

Reply:
done